README.md

```diff
@@ -81,6 +84,13 @@ To quickstart development, check out the [example folder](example) for a simple
 
 The `build.ps1` PowerShell script creates an empty webxcel project in `build/webxcel.xlsm`, which you can alter to create your table schema. You can also import the classes and modules by hand, but this will consume considerably more time since the import dialog only allows you to select one file at a time.
 
+In order for the build script to succeed, you may have to make a few changes to your Excel settings. Open the Trust Center in the options menu, select Macro Settings and check the following options:
+- Enable all macros
+- Trust access to the VBA project object model
+
+If you can't run the build script at all, use the following command to temporarily enable PowerShell scripts for the duration of your PowerShell session:
+```Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope Process```

```

Comment:
GitHub can't really handle that triple-backtick suggestion, but syntax highlighting would be nice 😄 

```suggestion
```powershell
Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope Process
```
```